backend/src/test/java/de/sambam/gamenighttracker/controller/UserControllerTest.java

```diff
@@ -170,11 +187,14 @@ public void getPlayerListTest() {
     public void addNewGameTest() {
         //GIVEN
         Game gameToAdd = Game.builder().name("Tabu").releaseYear("1990").build();
-        HttpEntity<Game> requestEntitiy = new HttpEntity<>(gameToAdd);
 
         //WHEN
+        String jwtToken = logintoApp();
+        HttpHeaders header = new HttpHeaders();
+        header.setBearerAuth(jwtToken);
+        HttpEntity<Game> entity = new HttpEntity<>(gameToAdd, header);
         ResponseEntity<Game> postResponse = testRestTemplate.exchange(
-                "http://localhost:" + port + "user/game", HttpMethod.POST, requestEntitiy, Game.class);
+                "http://localhost:" + port + "user/game", HttpMethod.POST, entity, Game.class);
```

Comment:
U could verify, that the game is in the db now too.

Reply:
done ^^'